tests/integration-tests/Apple/TestAppBundle.proj

```diff
@@ -14,7 +14,8 @@
 
   <PropertyGroup>
     <AppStorageUrl>$(AssetsBaseUri)/ios/test-app/$(TestTarget)</AppStorageUrl>
-    <XHarnessTestAppBundleUrl>$(AppStorageUrl)/$(TestAppBundleName).zip</XHarnessTestAppBundleUrl>
+    <!-- TODO: Revert -->
+    <XHarnessTestAppBundleUrl>$(AppStorageUrl)/$([System.String]::Copy($(TestAppBundleName)).Replace('.app', '.zip'))</XHarnessTestAppBundleUrl>
```

Comment:
Why do we need to change the `XHarnessTestAppBundleUrl`?

If we need to change the app, we should reach out to the helix team and they can re-upload a new version if necessary.

Reply:
@premun may have more context, but whatever doesn't work we'll most likely have to handle.

Reply:
Milos reached out to us last week so maybe this is just him testing the new ones?